client/container_inspect_test.go

```diff
@@ -49,11 +49,20 @@ func TestContainerInspectWithEmptyID(t *testing.T) {
 			return nil, errors.New("should not make request")
 		}),
 	}
-	_, _, err := client.ContainerInspectWithRaw(context.Background(), "", true)
+
+	_, err := client.ContainerInspect(context.Background(), "")
+	assert.Check(t, is.ErrorType(err, cerrdefs.IsInvalidArgument))
+	assert.Check(t, is.ErrorContains(err, "value is empty"))
```

Comment:
I should probably add an assert utility for this; we have a bunch of tests testing for this. Leaving that for a future exercise.